app/views/api/v2/locations/create.json.rabl

```diff
@@ -0,0 +1,3 @@
+object @taxonomy
+
+extends "api/v2/taxonomies/create"
```

Comment:
The response to create and update was apparently a default one (showing all the db fields) since it couldn't find the create/update rabl because there was none for the resource_id (location/organization).
This can be fixed in the controller by changing from `process_response` to an `if success/fail` and rendering the correct templates. I'm not sure which way is better